doc-content/enterprise-only/admin-and-config/business-central-editing-data-sources-proc.adoc

```diff
@@ -6,65 +6,59 @@
 From the *Data Source Authoring* page you can add data sources and database drivers to {CENTRAL}.
 
 [float]
-==== Adding a data source to {CENTRAL}
+== Adding a data source to {CENTRAL}
 
```

Comment:
Yes, some of these procedures should have introductory paragraphs. I'll see what I can do to add introductions.

The team uses "Procedure" headings rather than lead-in sentences or infinitive phrases on IBMSG p85.

Reply:
Added introductory paragraphs to related procedures